src/hooks/useFormSubmit.js

```diff
@@ -35,12 +35,12 @@ export default function useFormSubmit(
         if (error) {
           console.log(error);
         } else {
-          setSubmitted(true);
+          setIsFormSubmitted(true);
           console.log("Data saved successfully!");
         }
       });
     }
-
+    console.log(values);
     setFormValues(values);
```

Comment:
@sandrina-p this is where I clear the form by setting the state back to initial values (will rename it). I thought ```reset()``` doesn't work for controlled forms, ```formRef.current.reset()``` didn't work